native/cocos/platform/apple/modules/CanvasRenderingContext2DDelegate.mm

```diff
@@ -211,12 +215,17 @@ - (UIFont *)_createSystemFont {
 
 #endif
 
-- (void)updateFontWithName:(NSString *)fontName fontSize:(CGFloat)fontSize bold:(bool)bold {
+- (void)updateFontWithName:(NSString *)fontName fontSize:(CGFloat)fontSize bold:(bool)bold italic:(bool)italic {
     _fontSize = fontSize;
     _bold = bold;
+    _italic = italic;
 
     self.fontName = fontName;
     self.font = [self _createSystemFont];
+    float obliqueValue = 0.f;
+    if (_italic) {
+        obliqueValue = 0.25f;
```

Comment:
Hardcode ？

Reply:
Same as Android logic, Android`s suggest: for approximating oblique text, use values around 0.25